### PR TITLE
build: use cache when building from dockerfile

### PR DESCRIPTION
**Pull request by njelich**
This PR optimizes the build process slightly, making local development easier. Would be great if I could get some help with docker-sync or volumes setup on this. New to the Laravel dev loop.

### CODE REVIEW

**Comment by djaiss** [comment]
@asbiin is the master here about this. I don't know nothing about everything infra related. I'm a noob.

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "format": "prettier --write '**/*.{js,vue,css,scss}'",
     "pretest": "DB_CONNECTION=testing php artisan migrate:fresh && DB_CONNECTION=testing php artisan db:seed",
     "posttest": "vendor/bin/phpstan analyse && vendor/bin/psalm",
-    "test": "vendor/bin/phpunit"
+    "test": "vendor/bin/phpunit",
+    "docker:build": "docker build --cache-from monica-next -t monica-next -f scripts/docker/Dockerfile . && docker image prune -f",
```
Comment (asbiin) [inline package.json:11]: `docker image prune -f` will prune all images, it's not necessary in this case

**Review by asbiin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (njelich) [reply package.json:11]: It prunes only `<none>` images, I found it useful in terms of cleanup.

Comment (asbiin) [reply package.json:11]: OK, make sense.

**Pull request merged**